e2e/ang-e2e/tests/ang.spec.ts

```diff
@@ -0,0 +1,46 @@
+import {
```

Comment:
workspace.json doesn't look like it has an ang-e2e project? 

Please merge the specs into e2e/nx-adsp-e2e, and remove the extraneous files.